testprojects/tests/python/pants/dummies/BUILD

```diff
@@ -2,58 +2,3 @@ python_tests(
   name = 'passing_target',
```

Comment:
We still keep this target because it's used by `test_owners_integration_test.py`.